webpack/assets/javascripts/react_app/redux/actions/common/forms.test.js

```diff
@@ -49,31 +41,23 @@ describe('form actions', () => {
       });
   });
   it('on failed response handle base errors', () => {
-    nock('http://localhost')
-      .post('/api/resource')
-      .reply(404, { message: 'not found' });
-
     const store = mockStore({ resources: [] });
 
     return store
-      .dispatch(submitForm({ values: { a: 1 }, url: 'http://localhost/api/resource', item: 'Resource' }))
+      .dispatch(submitForm({ values: { a: 1 }, url: '/api/404', item: 'Resource' }))
       .then(() => {
         throw new Error('Should not hit this then block - test was set up incorrectly');
       })
       .catch((error) => {
         expect(error).toBeInstanceOf(SubmissionError);
-        expect(error.errors).toEqual({ _error: ['Error submitting data: 404 Not Found'] });
+        expect(error.errors._error[0]).toMatch(/Error submitting data: 404*/);
```

Comment:
can we get the actual error code + text?

Reply:
it looks like  `axios mock adapter`does not return `statusText` property.